cypress/cypress/integration/setup.spec.ts

```diff
@@ -167,14 +237,14 @@ describe("Setup", () => {
       cy.get("[data-cy=email]").within(($input) => {
         cy.get("input").should("have.value", "");
       });
-      cy.get("[data-cy=save-btn]").should("be.disabled");
     });
-    cy.get("[data-cy=radio-1]").should("have.css", "color", "rgb(255, 0, 0)");
+    cy.get("[data-cy=radio]")
+      .eq(1)
+      .should("have.css", "color", "rgb(255, 0, 0)");
     // fill out first name and save
+    cy.wait(500);
```

Comment:
makes me nervous that we have `cy.wait`...is here to address test flake? Is there no other way?

Reply:
Yeah, the issue is the slide plugin we're using has an animation that, during which, the buttons below lose focus so cypress can't click them